### PR TITLE
chore(vue): add storybook knobs

### PR DESCRIPTION
**Pull request by natashadecoste**
### Updates
- knobs for the vue stories (data, options)

### Demo screenshot or recording
image

### Review checklist (for reviewers only)
- [ ] Demos all features
- [ ] Documented/annotated
- [ ] Matches UI/UX specs
- [ ] Meets the [code style guide](https://github.com/IBM/carbon-charts/wiki/Code-style-guide)
- [ ] Accessible
- [ ] Mobile first (responsive)
- [ ] RTL support (bidirectional text)
- [ ] Performant (limited bloat)

### CODE REVIEW

File: packages/vue/.storybook/config.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
-setOptions({
+withOptions({
   name: 'Carbon Charts - Vue Wrappers',
-  showDownPanel: false,
-  showAddonPanel: false
+  sortStoriesByKind: true,
+  showDownPanel: true,
+  showAddonPanel: true,
+  panelPosition: 'bottom',
```
Comment (theiliad) [inline packages/vue/.storybook/config.js]: extra comma

**Comment by natashadecoste** [comment]
> Also changing any of the data or option knobs don't result in anything.
> 
> Is that a storybook problem or vue wrappers problem?

Looking into it now

**Comment by natashadecoste** [comment]
> Also changing any of the data or option knobs don't result in anything.
> 
> Is that a storybook problem or vue wrappers problem?

Seems others are having a problem with knobs when using data() for vue components as opposed to props. It seems to work and update nicely now with knobs, I'm not sure if you had a reason for using data() instead of props, let me know if that is the case.

**Comment by theiliad** [comment]
> > Also changing any of the data or option knobs don't result in anything.
> > Is that a storybook problem or vue wrappers problem?
> 
> Seems others are having a problem with knobs when using data() for vue components as opposed to props. It seems to work and update nicely now with knobs, I'm not sure if you had a reason for using data() instead of props, let me know if that is the case.

I don't have enough vue domain knowledge to decide that. As long as we're able to update the knobs and get the chart re-rendered then it works

**Pull request merged**